.env.sample

```
@@ -26,7 +26,7 @@ MAILGUN_FROM=
 ANTHROPIC_API_KEY=
 GOOGLE_GENERATIVE_AI_API_KEY=
 PLAUSIBLE=false
-SENTRY_DSN=  # web-next picks this up at build time (client bundle) and runtime (server)
+SENTRY_DSN=  # web-next injects this at runtime for the server and client
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)